controller/tenants_test.go

```diff
@@ -190,6 +190,34 @@ func (s *TenantsControllerTestSuite) TestSuccessfullyDeleteTenants() {
 			HasNoNamespace()
 	})
 
+	s.T().Run("ok when unsupported namespaces exist in DB", func(t *testing.T) {
+		// given
+		defer gock.OffAll()
+		testdoubles.MockCommunicationWithAuth(test.ClusterURL)
+		gock.New(test.ClusterURL).
+			Delete("/oapi/v1/projects/foo-che").
+			SetMatcher(test.ExpectRequest(test.HasJWTWithSub("devtools-sre"))).
+			Reply(200).
+			BodyString(`{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Success"}`)
+		gock.New(test.ClusterURL).
+			Delete("/oapi/v1/projects/foo").
+			SetMatcher(test.ExpectRequest(test.HasJWTWithSub("devtools-sre"))).
+			Reply(200).
+			BodyString(`{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Success"}`)
+
+		fxt := tf.FillDB(t, s.DB, tf.AddSpecificTenants(tf.SingleWithName("foo")),
```

Comment:
ok, so in this test, you fill the DB with records for some `user`, `che`, `jenkins`, `stage` and `run` tenants but only `user` and `che` will be deleted? All other namespaces/tenants are assumed to be already deleted (ie, `jenkins`, `stage` and `run`)? And at the end of the test, all records were removed from the DB?

Reply:
Yes, exactly :-)

Reply:
ok, thanks for clarifying ;)